src/client/PortalWebView.ts

```diff
@@ -130,53 +126,65 @@ export class PortalWebView {
 
     // Add styles to the current HTML so that it is displayed correctly in VS Code
     private addStyles(webview: vscode.Webview, html: string): string {
-        const uri =
-            vscode.workspace.workspaceFolders &&
-            vscode.workspace.workspaceFolders[0].uri;
-
-        // Add bootstrap.min.css
-        let url = webview.asWebviewUri(
-            vscode.Uri.joinPath(
-                uri as vscode.Uri,
-                "web-files",
-                "bootstrap.min.css"
-            )
-        );
-        const bootstrap = `<link href="${url}" rel="stylesheet" />`;
-
-        // Add theme.css
-        url = webview.asWebviewUri(
-            vscode.Uri.joinPath(uri as vscode.Uri, "web-files", "theme.css")
-        );
-        const theme = `<link href="${url}" rel="stylesheet" />`;
-
-        return html + bootstrap + theme;
+        const uri = PortalWebView.getPortalRootFolder();
+        if (uri) {
+            // Add bootstrap.min.css
+            let url = webview.asWebviewUri(
+                vscode.Uri.joinPath(
```

Comment:
consistency nit: most other .joinPath() are single line, which I think is more readable (assuming the # of args is low)

Reply:
fixed